daemon/execdriver/lxc/driver.go

```diff
@@ -87,6 +87,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		"-n", c.ID,
 		"-f", configPath,
 	}
+
+	// From lxc>=1.1 the default behavior is to daemonize containers after start
+	if d.version() >= "1.1" {
```

Comment:
both approaches are similar enough. the version comparison though, may eventually have additional cases where greater X has the option, but less than Y. Though I don't care to add additional execs, if we did check for the presence of the flag being used, then we really don't care _which_ version is being used ...

```
cmd := exec.Command("lxc-start", "--help")
  out, err := cmd.CombinedOutput()
  if err != nil {
    log.Fatal(err)
  }
  flag := []byte("--foreground")
  if bytes.Contains(out, flag) {
//...
```